pkg/codec/codec.go

```diff
@@ -23,7 +23,7 @@ import (
 var (
 	tablePrefix  = []byte{'t'}
 	metaPrefix   = []byte{'m'}
-	recordPrefix = []byte{'r'}
+	recordPrefix = []byte("_r")
```

Comment:
Is it necessary to change that?

Reply:
yes, it's same as TiDB. https://github.com/pingcap/tidb/blob/master/tablecodec/tablecodec.go#L45-L47